BusyList/Handlers/HelpHandler.cs

```diff
@@ -0,0 +1,49 @@
+﻿using BusyList.Commands;
+using BusyList.HelpSystem;
+using System;
+
+namespace BusyList.Handlers
+{
+    public class HelpHandler : IHandler<HelpCommand>
+    {
+        private readonly HelpProvider _helpProvider;
+
+        public HelpHandler(HelpProvider helpProvider)
+        {
+            _helpProvider = helpProvider;
+        }
+
+        public void Help()
```

Comment:
Could this be sourced from the gathered help attributes instead of a hardcoded list?

Reply:
wdym?

Reply:
I mean, instead of hardcoding a list of commands and their descriptions... isnt that exactly what we have in the helpprovider? Can we use that?

Reply:
you mean the help method in the helphandler?

Reply:
Yes. Currently, that method is just a hardcoded list of commands and descriptions. We should avoid that, so when a new command is added the help system automatically knows about it and can offer help.